host/sgx/ocalls/ocalls.c

```diff
@@ -397,8 +396,41 @@ oe_result_t oe_verify_quote_ocall(
         qe_identity_size,
         p_qe_identity_issuer_chain,
         qe_identity_issuer_chain_size);
+}
 
-    return result;
+oe_result_t oe_verify_quote_ocall_v2(
```

Comment:
Rename to `oe_verify_quote_ocall_tdx`/`oe_verify_quote_ocall_sgx`?

Reply:
Still work in progress